src/main/java/eu/dissco/exportjob/ProjectRunner.java

```diff
@@ -27,5 +27,6 @@ public void run(String... args) {
       log.error("An error has occurred", e);
     }
     context.close();
+    System.exit(0);
```

Comment:
Ok I believe the reason you need this is because of Spring Actuator. Actuator starts up a server for the health APIs. I wonder if we really need these health endpoints. Do you use them or do you see a use for them? I essence this is just a job like the translator and MAS which posts its status to the backend and runs until finished. The MAS and translator have no endpoints for health checks as we will never run health checks on them as they have a reason to go down at any moment.

Reply:
Thanks for the reply, that would make sense, but I don't use spring actuator in this project. It's only used  in the exporter backend. Is there another reason this project would need a System.exit()?